unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs

```diff
@@ -1,12 +1,15 @@
 using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 
 namespace DCL
 {
     public class AvatarsLODController : IAvatarsLODController
     {
         internal const float SIMPLE_AVATAR_DISTANCE = 10f;
+        private const int DEFAULT_MAX_AVATAR_VISIBLE = 70;
```

Comment:
I think we should use the `DataStore.i.avatarsLOD.maxNonLODAvatars.Get()` value that is already bound to a settings panel slider (we can rename it and/or change its defaults if you want) otherwise that setting would become useless now.
Renaming or changing it would involve modifying the following files:
* `unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsControllers/SpecificControllers/MaxNonLODAvatarsControlController.cs`
* `unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Configuration/Controls/MaxNonLODAvatarsControlConfiguration.asset`
* `unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Resources/Controls/MaxNonLODAvatarsControlController.asset`

Reply:
I didnt even know that data existed already, gonna change it, of course.

Nice catch!